integration/test/chartconfig/chartvalues/chartvalues_test.go

```diff
@@ -9,8 +9,8 @@ import (
 
 	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
 
-	"github.com/giantswarm/chart-operator/integration/chart"
-	"github.com/giantswarm/chart-operator/integration/chartconfig"
+	"github.com/giantswarm/chart-operator/integration/cnr"
+	"github.com/giantswarm/chart-operator/integration/customresource"
```

Comment:
Wasn't the specific naming better when there are multiple different custom resources?

Reply:
`chart` is renamed to `cnr`. So yes that works. OK I'll go with that.

Reply:
Done. @xh3b4sd PTAL.